scripts/build.ps1

```diff
@@ -347,6 +347,10 @@ Warn "Forcing content_handlers to $content_handlers"
 $adu_log_dir = '/tmp/aduc-logs'
 Warn "Forcing adu_log_dir to $adu_log_dir"
 
+# TODO(JeffMill): Temporary during development
```

Comment:
assuming there is something tracking to remind for removal; alternatively can remove